custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1247,6 +1247,24 @@
           "type": "property_name",
           "property_name": "child_name"
         }
+      },
+      {
+        "column_id": "location_reassignment_date",
+        "datatype": "date",
+        "type": "raw",
+        "property_name": "location_reassignment_date"
+      },
+      {
+        "column_id": "last_owner_id",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "location_reassignment_last_owner_id"
```

Comment:
I thought the case property name was kind of unwieldy, and this would be a little clearer. Plus they look kind of similar if you leave the whole thing since a multiple would have the form `location_reassignment_last_X_id`. I think the differences are a little clearer this way.